ines/36.js

```diff
@@ -0,0 +1,20 @@
+//1
+const student = { 
+    firstName: "John", 
+    lastName: "Smith", 
+    class: 12 };
+
```

Comment:
Task 1 ==> Create a METHOD that prints the following, therefore not just  using console.log()
So in this example the solution should be like this ==>
const student = {
  firstName: "John",
  lastName: "Smith",
  class: 12,
  printDetails() {
    return `${this.firstName} ${this.lastName} is in class ${this.class}.`;
  }
};
console.log(student.printDetails());